package.json

```diff
@@ -19,11 +19,10 @@
     "url": "https://github.com/csci4950tgt/honeyclient/issues"
   },
   "homepage": "https://github.com/csci4950tgt/honeyclient#readme",
+  "engines": {
+    "node": ">12.0.0"
```

Comment:
so we'll require the user to have a node ver > 12.0.0?

Reply:
should we throw that in the readme? Or does it auto-prompt?

Reply:
I should test that... 😅

Reply:
Looks like it doesn't prompt, the user has to configure the warning (what's the point of having it then, lol). I'll add something to the README as part of this PR.

Reply:
Fixed in `d674e145`.